src/latex_formats.jl

```diff
@@ -161,7 +161,17 @@ function pretty_latex_stats(io::IO, df::DataFrame;
 
   # by default, PrettyTables wants to boldface headers
   # that won't work if we put any math headers
-  tf = LatexTableFormat(latex_simple; header_envs=[])
+  tf = LatexTableFormat(
+    top_line       = "\\hline",
+    header_line    = "\\hline",
+    mid_line       = "\\hline",
+    bottom_line    = "\\hline",
+    left_vline     = "|",
+    mid_vline      = "|",
+    right_vline    = "|",
```

Comment:
Visually, I find tables look much better without vertical lines. Will this force vertical lines?

Reply:
I just kept the default parameters as before, only the constructor has changed here.
On the two examples from the unit tests there are no vertical lines.

Reply:
Ok thank you for checking!